@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -529,6 +538,40 @@ const deleteItem = <T,>(array: T[], index: number) => {
 
 const poolName = computed(() => pool.value?.name_label)
 
+const poolCpuInfo = computed(() => {
+  return {
+    cpus: {
+      cores: pool.value?.cpu_info && +pool.value.cpu_info.cpu_count,
+      sockets: pool.value?.cpu_info && +pool.value.cpu_info.socket_count,
+    },
+  }
```

Comment:
Is there a reason to return an object with a single key, instead of this?
```ts
return {
  cores: pool.value?.cpu_info && +pool.value.cpu_info.cpu_count,
  sockets: pool.value?.cpu_info && +pool.value.cpu_info.socket_count,
}
```